flames/selenium/docker-compose.override.yml

```diff
@@ -0,0 +1,52 @@
+version: '2.1'
+
+services:
+
+    selenium-hub:
```

Comment:
I would keep this shorter and just use `selenium`. Or do you see any naming collisions?

Reply:
There are no reason that we cannot change it from `selenium-hub` to `selenium`. But be aware that it may be confused with nodes (chrome and firefox)